lib/compat/wordpress-5.9/class-wp-theme-json-5-9.php

```diff
@@ -179,6 +179,7 @@ class WP_Theme_JSON_5_9 {
 		'padding-right'              => array( 'spacing', 'padding', 'right' ),
 		'padding-bottom'             => array( 'spacing', 'padding', 'bottom' ),
 		'padding-left'               => array( 'spacing', 'padding', 'left' ),
+		'position'                   => array( 'layout', 'position' ),
```

Comment:
Should this be in the `compat/6.0/class-wp-theme-json-gutenberg.php`

Haven't checked if this branch has been rebased yet.

```php
class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_Gutenberg_5_9 {
	/**
	 * The valid properties under the settings key.
	 *
	 * @var array
	 */
	const VALID_SETTINGS = array(
		...
		'layout'          => array(
			'contentSize' => null,
			'position'    => null,
			'wideSize'    => null,
		),
		...
        );
}
```

Reply:
See thread above https://github.com/WordPress/gutenberg/pull/38039#discussion_r808702086